src/System.Windows.Forms.Analyzers/src/System/Windows/Forms/Analyzers/Diagnostics/DiagnosticIDs.cs

```diff
@@ -5,6 +5,8 @@ namespace System.Windows.Forms.Analyzers.Diagnostics;
 
 internal static class DiagnosticIDs
 {
+    public const string UrlFormat = "https://aka.ms/winforms-experimental/{0}";
```

Comment:
Sorry I missed this. We should make this more generic like Runtime does:

```suggestion
    public const string UrlFormat = "https://aka.ms/winforms-warnings/{0}";
```

Reply:
My thinking is that we later have more AKAs, like winforms-experimental and winforms-warnings and ... so we can have more landing pages for categories.  But I have no preference. If you want this all warnings, I can create a new alias.

Reply:
Let's start with following the libraries pattern.

Reply:
NB: now you have two copies of the same const:
![image](https://github.com/user-attachments/assets/7c6e19e2-aaeb-4981-a5a1-5a1ae06d2989)

Reply:
@RussKie thanks for pointing this out. We'll clean it up.